plone/app/linkintegrity/browser/info.py

```diff
@@ -55,7 +55,10 @@ def get_breaches(self, items=None):
             # list of uids that are ignored
             uids_to_ignore.extend([i.UID for i in brains_to_delete])
             for brain_to_delete in brains_to_delete:
-                obj_to_delete = brain_to_delete.getObject()
+                try:
+                    obj_to_delete = brain_to_delete.getObject()
+                except (AttributeError, KeyError):
+                    continue
```

Comment:
This would probably indicate that there are out of date catalog records (i.e. wrong allowedUsersAndRoles, or an entry for an object that no longer exists), so maybe we should log something?